bika/lims/browser/analysisrequest/add2.py

```diff
@@ -6,30 +6,28 @@
 # Some rights reserved. See LICENSE.rst, CONTRIBUTORS.rst.
 
 import json
-import magnitude
 from datetime import datetime
-from DateTime import DateTime
 
+import magnitude
 from BTrees.OOBTree import OOBTree
-
+from DateTime import DateTime
+from Products.CMFPlone.utils import _createObjectByType
+from Products.CMFPlone.utils import safe_unicode
+from Products.Five.browser import BrowserView
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
 from plone import protect
-
-from plone.memoize.volatile import cache
 from plone.memoize.volatile import DontCache
-
+from plone.memoize.volatile import cache
 from zope.annotation.interfaces import IAnnotations
-from zope.publisher.interfaces import IPublishTraverse
-from zope.interface import implements
+from zope.component import queryAdapter
 from zope.i18n.locales import locales
-
-from Products.CMFPlone.utils import safe_unicode
-from Products.CMFPlone.utils import _createObjectByType
-from Products.Five.browser import BrowserView
-from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from zope.interface import implements
+from zope.publisher.interfaces import IPublishTraverse
 
 from bika.lims import api
-from bika.lims import logger
 from bika.lims import bikaMessageFactory as _
+from bika.lims import logger
+from bika.lims.interfaces import IGetDefaultFieldValueARAddHook
 from bika.lims.utils import tmpID
 from bika.lims.utils.analysisrequest import create_analysisrequest as crar
```

Comment:
Here also a separate commit please

Reply:
What do you mean? A separate commit for import modifications?

Reply:
Ok